client/src/components/Login/index.tsx

```diff
@@ -1,19 +1,40 @@
-import React from 'react'
+import React, {useEffect} from 'react'
 import { useQuery } from 'react-apollo'
 import { GET_USERS } from './gql'
 
 import './login.scss'
 import { FaGithubSquare } from 'react-icons/fa'
 import { FaFacebookSquare } from 'react-icons/fa'
+import axios from 'axios'
+import queryParser from 'query-parser-url'
+const baseUrl = 'http://localhost:4000'
+const githubLoginUrl = 'https://github.com/login/oauth/authorize'
+const githubCallbackUrl = `http://localhost:3000`
+const GITHUB_CLIENT_ID='9368cfd68ad4ee3c646e'
 
 export const Login = () => {
+  useEffect(() => {
+    const url = window.location.search.substring(1);
+
+    const queryObject = queryParser(url)
+    const code = queryObject['code']
+    if(!code) return
+
+    axios.get(`/api/github-login/?code=${code}`).then(res => {
+      localStorage.setItem('token', res.data.token)
+    })
+  })
+  const githubLoginHandler = async e => {
+    location.href = `https://github.com/login/oauth/authorize/?client_id=${GITHUB_CLIENT_ID}&redirect_uri=${githubCallbackUrl}`
+  }
+
   return (
     <div className="login-section">
       <img className="logo" src="./images/bmartlogo.jpeg" />
       <div className="social-login-wrap">
         <div className="login-btn-wrap">
           <FaGithubSquare size={50} />
-          <div className="login-text">Login woth Github</div>
+          <div className="login-text" onClick={githubLoginHandler}>Login with Github</div>
```

Comment:
여기에 a 태그를 사용하는 건 어떻게 생각하시나요?
클릭이벤트에 추가될만한 동작이 없으면 a태그를 사용해도 괜찮아보입니다!

```suggestion
         const githubLoginHref = `https://github.com/login/oauth/authorize/?client_id=${GITHUB_CLIENT_ID}&redirect_uri=${githubCallbackUrl}`
         
          <a className="login-text" href={githubLoginHref}>Login with Github</a>
```

Reply:
네 좋은거 같아요! 저도 생각했는데 영지님 프론트 작업이랑 겹칠까봐 일단 이벤트 핸들러로 동작하게 해놓았습니다.

나중에 머지하고 a 태그 붙여도 될거 같아요!